voice.go

```diff
@@ -787,7 +785,8 @@ func (v *VoiceConnection) opusSender(udpConn *net.UDPConn, close <-chan struct{}
 			sequence++
 		}
 
-		if (timestamp + uint32(size)) >= 0xFFFFFFFF {
```

Comment:
The value given is greater than the max value of Uint32 which is impossible. Instead, we should check for integer ovrflow.